markdown/4.x/en/documentation/single-hit-redirect.md

```diff
@@ -1,10 +1,10 @@
 ## Single Hit Redirect
 
-As is mentioned in the `ff-communication` [API reference](/api/4.x/ff-communication#tab=api), the built-in single hit redirect feature, due to its limitations, is now deprecated and will be removed in a future release.
+The built-in single hit redirect feature, due to its limitations, was deprecated in version 3.x and since 4.x is removed.
```

Comment:
Version numbers tend to be formatted in monospace font.
```suggestion
The built-in single hit redirect feature, due to its limitations, was deprecated in version `3.x` and since `4.0.0` it is no longer available.
```